libraries/protocol/include/graphene/protocol/asset_ops.hpp

```diff
@@ -392,13 +409,19 @@ namespace graphene { namespace protocol {
     */
    struct asset_publish_feed_operation : public base_operation
    {
+      struct ext
+      {
+         /// After BSIP77, price feed producers can feed ICR too
```

Comment:
Please note that the current spec for BSIP77 describes that asset owners can update the ICR.  But it makes no mention about feed producers.

Reply:
Yes I'm aware that it's not in the specification. But I think it makes sense so I implemented it anyway.

Reply:
Fair enough.  It does it make it consistent with how MCR and MSSR are handled.